packages/@cdklabs/cdk-cicd-wrapper/src/stacks/VPCStack.ts

```diff
@@ -38,9 +58,33 @@ export class VPCStack extends cdk.Stack {
    */
   readonly vpc: ec2.IVpc | undefined;
 
+  /**
+   * The security group attached to the VPC
+   */
+  securityGroup: ec2.ISecurityGroup | undefined;
+
+  /**
+   * The subnets attached to the VPC
+   */
+  subnetType: ec2.SubnetType | undefined;
```

Comment:
please use the question mrk syntax, I don't know why here we have this undifined union. This has to be readonly as well, right?